src/components/ContentHeader.vue

```diff
@@ -99,13 +98,14 @@ div.usa-nav-container {
 .usa-nav__primary, .usa-accordion {
  align-items: right;
  float: right;
+ background-color: $nearBlack;
 }
 .usa-header.usa-header--basic {
```

Comment:
This is quite tall on the mobile preview, and the sticky part blocks a lot of the view when scrolling. Might eventually want to make smaller - maybe even make it a hamburger menu on mobile?